src/components/Sort.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+
+const Sort = ({ className, setOrderBy }) => {
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
💬 여담
reducer 를 사용해 아래처럼 간단하게도 작성 가능합니다~

```suggestion
const [isOpen, toggleIsOpen] = useReducer((prev) => !prev, false);
```